cmd/daemon/serve.go

```diff
@@ -234,7 +234,7 @@ func sqa(ctx stdctx.Context, cmd *cobra.Command, d driver.Registry) *metricsx.Se
 		d.Config().GetProvider(ctx),
 		&metricsx.Options{
 			Service:       "ory-kratos",
-			ClusterID:     metricsx.Hash(d.Persister().NetworkID(ctx).String()),
+			DeploymentId:  metricsx.Hash(d.Persister().NetworkID(ctx).String()),
```

Comment:
There are two other new options `StartTime` and `DBDialect`
https://github.com/ory/x/compare/v0.0.552...v0.0.557#diff-a2edd5b012443ff9112a3ca86e7380bef9f9dfeeda39e83db2b059132b37746fR71-R77

I've have not set them here, if I should change that, I'm happy to do it :)
Not sure what the intention behind those was